sirepo/job_driver/__init__.py

```diff
@@ -194,6 +203,7 @@ async def terminate(cls):
 
     def websocket_on_close(self):
         pkdlog("{}", self)
+        self._websocket = None
```

Comment:
The next line starts free resources, which is async. The _websocket is invalid at the point of websocket_on_close so it has to be invalided in case other functions try to use it.